src/__tests__/index.spec.js

```diff
@@ -232,10 +232,11 @@ describe('afterEmitHook', () => {
     const sentryCliPlugin = new SentryCliPlugin({
       include: 'src',
       release: '42',
-      commit: '4d8656426ca13eab19581499da93408e30fdd9ef',
-      previousCommit: 'b6b0e11e74fd55836d3299cef88531b2a34c2514',
-      repo: 'group / repo',
-      // auto,
+      setCommits: {
+        commit: '4d8656426ca13eab19581499da93408e30fdd9ef',
+        previousCommit: 'b6b0e11e74fd55836d3299cef88531b2a34c2514',
+        repo: 'group / repo',
+      },
     });
```

Comment:
I would suggest to add test for both versions of calling `SentryCliPlugin({ ..., commit, previousCommit, repo })` and `SentryCliPlugin({ ..., setCommits: { commit, previousCommit, repo } })`

Reply:
Thank you! I just used the "easy way" by using the patch posted below. Is there something you would like to add on that, or are we good? :D

Reply:
We are good 🚀